Formula/jsdoc3.rb

```diff
@@ -18,7 +18,10 @@ class Jsdoc3 < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+     inreplace libexec/"lib/node_modules/jsdoc/node_modules/requizzle/lib/requizzle.js",
+               "if (lookupPaths[0] === targetPath && lookupPaths[1].length === 0) {",
```

Comment:
https://github.com/Homebrew/homebrew-core/blob/1b23cc14a8162dd0bb59977b23a0e00c62941217/Formula/orientdb.rb#L17-L21

The existing similar code uses just 1 indent, instead of 5 indents.